src/spatial/collision.js

```diff
@@ -266,7 +258,7 @@ Crafty.c("Collision", {
      *
      * @see 2D
      */
-    hit: function (comp) {
+    hit: function (component) {
```

Comment:
Renamed as [this](https://github.com/craftyjs/Crafty/blob/develop/src/spatial/collision.js#L235)